thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -1,7 +1,7 @@
 import Component from "@ember/component";
 import { computed, setProperties, getProperties, get } from '@ember/object';
 import moment from 'moment';
-import { humanizeFloat } from 'thirdeye-frontend/helpers/utils';
+import { humanizeFloat, filterPrefix, toOffsetUrn } from 'thirdeye-frontend/helpers/utils';
```

Comment:
is humanizeFloat an IT reference? we'll all float one day...

Reply:
Haha I didn't write this, so I can't claim the credit, but that's a pretty cool reference.